master/buildbot/www/hooks/github.py

```diff
@@ -170,7 +187,21 @@ def handle_pull_request(self, payload, event):
 
         log.msg("Received {} changes from GitHub PR #{}".format(
             len(changes), number))
-        return changes, 'git'
+        defer.returnValue((changes, 'git'))
+
+    @defer.inlineCallbacks
+    def _get_commit_msg(self, repo, sha):
+        '''
+        :param repo: the repo full name, ``{owner}/{project}``.
+            e.g. ``buildbot/buildbot``
+        '''
+        url = '/repos/{}/commits/{}'.format(repo, sha)
+        http = yield httpclientservice.HTTPClientService.getService(
+            self.master, 'https://api.github.com')
```

Comment:
I want to leave it with your fabulous `GithubApiService` in PR #3404, so hard-coded this url is fine for me in this PR.

If you agree, I can add an TODO comment about that.